src/main/java/org/apache/cassandra/sidecar/restore/RestoreJobDiscoverer.java

```diff
@@ -264,7 +264,7 @@ private void findSlicesOfRangeAndSubmit(InstanceMetadata instance, RestoreJob re
     {
         short bucketId = 0; // TODO: update the implementation to pick proper bucketId
         restoreSliceDatabaseAccessor
-        .selectByJobByBucketByTokenRange(restoreJob.jobId, bucketId, range.start, range.end)
+        .selectByJobByBucketByTokenRange(restoreJob.jobId, bucketId, range.start().toBigInteger(), range.end().toBigInteger())
```

Comment:
Should we make `selectByJobByBucketByTokenRange` just accept a `TokenRange range` instead of a `range.start()` and a `range.end()`?